libkbfs/bare_root_metadata_v3.go

```diff
@@ -88,16 +88,19 @@ type BareRootMetadataV3 struct {
 	// of writing to the given folder.
 	FinalizedInfo *tlf.HandleExtension `codec:"fi,omitempty"`
 
-	// The sequence number of the global Keybase Merkle tree at the
-	// time this update was created (from the writer's perspective).
-	// This field was added to V3 after it was live for a while, and
-	// older clients that don't know about this field yet might copy
-	// it into new updates via the unknown fields copier. Which means
-	// new MD updates might end up referring to older Merkle roots.
-	// That's ok since this is just a hint anyway, and shouldn't be
-	// fully trusted when checking MD updates against the Merkle tree.
+	// DEPRECATED -- see KBMerkleRoot.Seqno instead.
```

Comment:
Oh good point. The unknown handler would populate the zero value, which is also defined as unknown, right? If so, sounds good to me!

> one other advantage to this way is that if other fields are added to MerkleRootV2 in the future, we'll get them for free.

Good point. Any updates to the `MerkleRootV2` would be backward compatible, so it's safe to have that also here!